README.md

```diff
@@ -14,60 +14,37 @@ cd monocle
 mkdir data
 ```
```

Comment:
chmod o+w data

w/o it elasticsearch does not start. Forbidden to write in data.

Reply:
ok changing that

Reply:
but it works for me without it btw...

Reply:
done